flowgate-common/src/main/java/com/vmware/flowgate/common/model/ValueUnit.java

```diff
@@ -10,12 +10,21 @@
 import com.vmware.flowgate.common.exception.WormholeException;
 
 public class ValueUnit {
+   private String id;//This id is used to identify a outlet of PDU
```

Comment:
It's not ID.  ValueUnit don't have ID.  we only need some extra field for some special cases.  For most of the case this field is empty.

id will case too much confusing. 

let's change it to  "extraidentifier"